build.sh

```diff
@@ -32,13 +32,13 @@ while getopts qe c; do
   \?) usage;;
   esac
 done
-shift `expr $OPTIND - 1`
+shift $( expr $OPTIND - 1 )
 
 if [ $# -eq 0 ]; then
   set -- core plugins gradle maven
 fi
 
-set -e  # exit on error
+set -o errexit  # exit on error
```

Comment:
Second newby question: what's the difference here as well?

Reply:
Same, `-e` is just a shortcut.